indexing/metadata.md

```diff
@@ -18,44 +18,199 @@ Including the following tags and metadata in each of your pages will improve the
+- **Used In:** Query matching, term frequency scoring
+
+### Tags & Keywords
+
+We look for the following fields:
+```
+<meta name="keywords" content="example, example 2, example 3" />
+<meta name="dcterms.keywords" content="example" />
+<meta name="dc.subject" content="example" />` 
+<meta name="dcterms.subject" content="example" />
+<meta name="article:tag" content="example" />
+<meta name="article:section" content="example" />
+```
+- **Detail:** While not often used by commercial search engines due to [keyword stuffing](https://support.google.com/webmasters/answer/66358?hl=en), Search.gov indexes your keywords, if you have added them. 
+- **Data Gathering Logic:** We pull all fields listed into one "Tags" field for searching and future facet filtering.
+- **Used In:** Query matching, term frequency scoring, faceted search (upcoming feature)
+
+
+### Created Date
+
+We look for the following fields:
+```
+<meta property="article:published_time" content="YYYY-MM-DD" />
+<meta name="dc.date" content="YYYY-MM-DD" />
+<meta name="dc.date.created" content="YYYY-MM-DD" />`  
+<meta name="dcterms.created" content="YYYY-MM-DD" />
+```
+- **Detail:** The date fields are listed in order of preference. We only accept one `created` date, and use this list to determine the fallback options. Exact time is optional; [read more here](https://en.wikipedia.org/wiki/ISO_8601).
+- **Used In:** Page freshness scoring, faceted search (upcoming feature)
+
+
+### Changed Date
+
+We look for the following fields:
+```
+<meta property="article:modified_time" content="YYYY-MM-DD" />
+<lastmod>
+```
+
+- **Detail:** Exact time is optional; [read more here](https://en.wikipedia.org/wiki/ISO_8601). The `<lastmod>` field is included in XML sitemaps to signal to search engines when a page was last modified. Search.gov collects this metadata in case there is no `article:modified_time` data included in the page itself.
+- **Used In:** Page freshness scoring, faceted search (upcoming feature)
+
+
+### Crawling Metadata
+
+We look for the following field:
+```
+<meta name="robots" content="..., ..." />
+```
+- **Detail:** Use the [meta robots tag]({{ site.baseurl }}/indexing/how-search-engines-index-content-better-discoverability.html#robots) to block the search engine from indexing a particular page.
+- **Used In:** Used during indexing, does not affect relevance ranking.
+
+
+
+
+## Upcoming Metadata Support
+The following fields will power new features coming in the next few months. These will not impact your search until we release those features. We recommend adding these fields to your documents *now* to use these new features sooner. 
+
+### Thumbnail Image
+
+We look for the following field:
+```
+<meta property="og:image" content="https://domain.com/my-image-url.jpg" />
+```
+- **Detail:** An image associated to the content. Ideally, this image serves well as a thumbnail on both our search results page and on social media websites like Facebook and Twitter.
```

Comment:
let's change "social media websites like Facebook and Twitter" to "social media platforms"

Reply:
fixed